modules/diskq/Makefile.am

```diff
@@ -1,18 +1,10 @@
-BUILT_SOURCES += modules/diskq/diskq-grammar.y modules/diskq/diskq-grammar.c modules/diskq/diskq-grammar.h
-EXTRA_DIST += $(BUILT_SOURCES) modules/diskq/diskq-grammar.ym
-
-module_LTLIBRARIES += modules/diskq/libdisk-buffer.la
 bin_PROGRAMS += modules/diskq/dqtool
+lib_LTLIBRARIES += modules/diskq/libsyslog-ng-disk-buffer.la
+module_LTLIBRARIES += modules/diskq/libdisk-buffer.la
```

Comment:
Do we really want to install this library? I think it is enough to create a static lib and link that to all users.

We don't want to deal with lib versioning or anything of that sort IMHO.